modules/ROOT/pages/flex-gateway-add.adoc

```diff
@@ -0,0 +1,20 @@
+= Add a Flex Gateway
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-disabled.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub"]
+image:logo-hybrid-active.png[xref="deployment-strategies.adoc#hybrid-deployments", title="Hybrid"]
+image:logo-server-disabled.png[xref="deployment-strategies.adoc#anypoint-platform-pce-deployments", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[xref="deployment-strategies.adoc#anypoint-runtime-fabric-deployments", title="Runtime Fabric"]
+
+To add a new Flex gateway, follow the instructions described in the UI:
```

Comment:
Should we always capitalize Flex Gateway? (and if just gateway, not specifying Flex, leave uncapitalized)

Throughout many of the pages, I see a mix of Flex Gateway, Flex gateway, flex gateway.

Reply:
For coherence, we'll capitalize only when referring to the product, in titles, or when referring to the UI (tabs, buttons).